packages/stanza-react/src/createStanzaInstance.ts

```diff
@@ -4,8 +4,12 @@ import { type StanzaInstance } from './stanzaInstance';
 
 export type StanzaConfig = StanzaCoreConfig;
 
-export const createStanzaInstance = (config: StanzaConfig): StanzaInstance => {
-  StanzaBrowser.init(config);
+export const createStanzaInstance = async (
+  config: StanzaConfig
+): Promise<StanzaInstance> => {
+  StanzaBrowser.init(config).catch((e) => {
+    console.warn('Error while initializing @getstanza/browser', e);
+  });
```

Comment:
this doesn't have to be async as we are not awaiting anything
```suggestion
export const createStanzaInstance = (
  config: StanzaConfig
): StanzaInstance => {
  StanzaBrowser.init(config).catch((e) => {
    console.warn('Error while initializing @getstanza/browser', e);
  });
```